swift_package/BrainFlowCI/BrainFlowCI.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
do we need this file in repo?

Reply:
I think it's not strictly necessary, but nice to have, per this post:

Reply:
I did a git rm --cached on this file.